src/components/Preview.js

```diff
@@ -34,6 +34,36 @@ function Preview({ markup, accessibleRoles, elements, dispatch }) {
 
   // TestingLibraryDom?.getSuggestedQuery(highlighted, 'get').toString() : null
 
+  const scripts = useMemo(() => {
+    const container = document.createElement('div');
+    container.innerHTML = markup;
+    const scriptsCollections = container.getElementsByTagName('script');
+    return Array.from(scriptsCollections).map((script) => script.innerHTML);
```

Comment:
This also parses `script` elements of other types than javascript. For example `<script type="text/css">`. Type attributes for `script` elements default to `text/javascript`, so we should only extract `script` elements without a `type` attribute, or with a `type === "text/javascript"`

```html
<script type="text/javascript">
  document.getElementById('btn').addEventListener('click', () => {
    alert('click');
  });
</script>
```

See also [this repl](https://deploy-preview-128--testing-playground-com.netlify.app/?markup=DwIwrgLhD2B2AEBLAJgXgEQgrdA%2BAUAIIDGxApgA4T7AD04UcBNAzsQE6JXwQCeFZDBDIAPCLQBWAQwBuUtpyp58yaMTABbMrAgA6AOZkIAUQA2ZLToBCvAJLIAFAHIssJwEpdU5MmMztEAAyiCzCsGTszsSmiMQA1k4ANPAO7vCouPAA3vhS5uwQUTHxHgDc%2BAC%2B7uV0ClwQuEA&query=M4YwTgphB2B0DmEAuAhAngJQPYBsIAoByAIwFckktpCAaAAgG8AoaAQwFsIAuOgelZAgIAByS8AlkwC%2BASiA).

Reply:
Got it! I am going to fix it again!